### PR TITLE
Breaking server.py 

### PR DESCRIPTION
**Pull request by viskey98**
This PR will break server.py. This was done with the base of latest rewritten server. After the re-written server gets merged, this pr will again be re-based if necessary. :)

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    def start(self):
+        app = web.Application()
+        app.add_routes([web.route('*', '/{tail:.*}', self.handle_request)])
+        aiohttp_jinja2.setup(
+            app, loader=jinja2.FileSystemLoader(self.dir)
+        )
+        middleware = SnareMiddleware(self.meta['/status_404']['hash'])
+        middleware.setup_middlewares(app)
+        web.run_app(app, host=self.run_args.host_ip)
```
Comment (afeena) [inline server.py]: and what about port?

File: server.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        self.sroute = StaticRoute(
+            name=None, prefix='/',
+            directory=self.dir
+        )
+        self.tanner_handler = TannerHandler(run_args, meta, snare_uuid)
+
+    async def handle_request(self, request):
+        self.logger.info('Request path: {0}'.format(request.path))
+        data = self.tanner_handler.create_data(request, 200)
+        if request.method == 'POST':
```
Comment (afeena) [inline server.py]: maybe we can split post and get handlers?

Comment (viskey98) [reply server.py]: what about PUT and other requests? If we split both handlers, we need to define handlers for others HTTP request as well. What do you think about it?

**Comment by viskey98** [comment]
Fixed by latest PR #141

**Pull request closed without merging**